.github/workflows/discord_updater.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       run: |
         git config --global user.name ${GITHUB_ACTOR}
         git config --global user.email ${GITHUB_ACTOR}@github.com
+        git checkout released
```

Comment:
It sounds safer to do this in the checkout part:
````
- uses: actions/checkout@v2
      with:
        fetch-depth: 0
        ref: develop
````
Otherwise, you apply the script on the master branch and then only switch branch (just to make sure to avoid any possible conflicts).

Reply:
Thank you! I guess you meant `ref: released` instead of `ref: develop`. I did that change.

Reply:
You're welcome, you're right, it is indeed `ref: released`.